appveyor.yml

```diff
@@ -42,3 +42,7 @@ build_script:
 test_script:
   - echo "%JL_TEST_SCRIPT%"
   - C:\julia\bin\julia -e "%JL_TEST_SCRIPT%"
+
+after_script:
```

Comment:
`on_success`? See https://www.appveyor.com/docs/build-configuration/

Reply:
Also, please make the same change to `README.md`

Reply:
Will do